src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1421,4 +1415,16 @@ private void OnSpeciesNameTextChanged(string newText)
             speciesNameEdit.Set("custom_colors/font_color", new Color(1, 1, 1));
         }
     }
+
+    private Dictionary<string, float> makeSortedConsumptionBar (Dictionary<string, float> consumptionBar)
+    {
+        Dictionary<string, float> result = consumptionBar;
+
+        result = result.OrderBy(
+            i => i.Key != "baseMovement" && i.Key != "osmoregulation")
```

Comment:
This sort should be fixed.
A sort function needs to work the same way either way, and this doesn't even sort the other keys.
https://stackoverflow.com/questions/985657/use-own-icomparert-with-linq-orderby